### DISCUSSION TITLE
add file pattern `requirements.*.txt` for `pip_requirements`

### DISCUSSION THREAD

**Discussion by trim21** [renovatebot/renovate#28126] [Suggest an Idea]
### Tell us more.

currently `pip_requirements` mgr only support pattern `requirements-*.txt` or `requirements-*.pip`, hope could also add ``

**Comment by rarkins** [comment]
Sure, PR welcome

**Reply by HonkingGoose** [reply]
This PR resolves this discussion:

- https://github.com/renovatebot/renovate/pull/28143

The changes from the PR landed in Renovate version `37.272.0`.
